components/published-toggle/index.jsx

```diff
@@ -16,6 +17,16 @@ const PublishedToggle = ({
     else toggletext = 'UnPublished'
   }
 
+  const handleChange = useCallback(async event => {
```

Comment:
Why do you use `useCallback` hook here?

Reply:
It's just a very small optimization so the function doesn't get recomputed on every rerender.